### PR TITLE
Amazon EKS based on Ubuntu 20.04 ARM64

### PR DESCRIPTION
**Pull request by amulobaev**
Added scripts and config files to create custom AMI based on Ubuntu 20.04 (ARM64, k8s 1.19).

Components:
- CNI plugin v0.9.1
- Docker 19.03.13
- containerd 1.4.6

Changes:
- Change cgroup driver to systemd
- Add SSD support (m6gd instance type)
- Install kernel module shiftfs via dkms

### CODE REVIEW

File: scripts/shared/docker.sh [hunk 1]
```diff
@@ -51,7 +51,7 @@ elif is_ubuntu; then
   curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add -
   add-apt-repository "deb [arch=${ARCH}] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
   apt-get update -y
-  apt-get install -y docker-ce docker-ce-cli containerd.io
+  apt-get install -y docker-ce docker-ce-cli containerd.io=1.4.6-1
```
Comment (pavan-tri) [inline scripts/shared/docker.sh:54]: Do we need versioning for docker-ce as well eg. 19.x or 20.x

File: scripts/shared/eks.sh [hunk 2]
```diff
@@ -134,10 +137,15 @@ systemctl daemon-reload && systemctl disable kubelet
 ################################################################################
 
 mkdir -p /etc/eks
-curl -sL -o /etc/eks/eni-max-pods.txt https://raw.githubusercontent.com/awslabs/amazon-eks-ami/master/files/eni-max-pods.txt
```
Comment (pavan-tri) [inline scripts/shared/eks.sh:137]: Max pods let it download from github

File: scripts/shared/eks.sh [hunk 3]
```diff
@@ -134,10 +137,15 @@ systemctl daemon-reload && systemctl disable kubelet
...
-curl -sL -o /etc/eks/bootstrap.sh https://raw.githubusercontent.com/awslabs/amazon-eks-ami/master/files/bootstrap.sh
+cp /etc/packer/files/eni-max-pods.txt /etc/eks/eni-max-pods.txt
+cp /etc/packer/files/bootstrap.sh /etc/eks/bootstrap.sh
```
Comment (pavan-tri) [inline scripts/shared/eks.sh:141]: How will we do fixes from bootstrap. Is it possible to mark our changes as another dsoverride.sh and call that from bootstrap ?

**Review by pavan-tri — CHANGES_REQUESTED** [review]

File: scripts/shared/eks.sh (hunk 3 above)
Comment (amulobaev) [reply scripts/shared/eks.sh:141]: Script bootstrap.sh will be updated only manually.
Original file from the base repo contains section 'gpu boost clock' which is not supported by Ubuntu AMI

**Review by pavan-tri — APPROVED** [review]

**Pull request merged**